Jenkinsfile

```diff
@@ -259,6 +259,61 @@ def k8sTest(Map args = [:]) {
   }
 }
 
+/**
+* This method runs the packaging
+*/
+def packagingLinux(Map args = [:]) {
+  def PLATFORMS = [ '+all',
+                'linux/amd64',
+                'linux/386',
+                'linux/arm64',
+                'linux/armv7',
+                // The platforms above are disabled temporarly as crossbuild images are
+                // not available. See: https://github.com/elastic/golang-crossbuild/issues/71
+                //'linux/ppc64le',
+                //'linux/mips64',
+                //'linux/s390x',
+                'windows/amd64',
+                'windows/386',
+                (params.macos ? '' : 'darwin/amd64'),
+              ].join(' ')
+  withEnv([
+    "PLATFORMS=${PLATFORMS}"
+  ]) {
+    target(args)
+  }
+}
+
+def e2e(Map args = [:]) {
+  def suite = args.get('suite', '')
+  def tags = args.get('tags', '')
+  def stackVersion = args.get('stackVersion', '8.0.0-SNAPSHOT')  // TBC with the version defined somewhere...
+  dir('.e2e') {
+    gitCheckout(
+      branch: "master",  // TBC with the target branch if running on a PR basis.
+      repo: "https://github.com/elastic/e2e-testing.git",
+      credentialsId: "${JOB_GIT_CREDENTIALS}"
+    )
+    try {
+      if(isInstalled(tool: 'docker', flag: '--version')) {
+        dockerLogin(secret: "${DOCKER_ELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
+      }
+      retry(3){
+        sh script: """.ci/scripts/install-test-dependencies.sh "${suite}" """, label: "Install test dependencies for ${suite}:${tags}"
+      }
+      filebeat(output: "docker_logs_${suite}_${tags}.log", workdir: "${env.WORKSPACE}"){
+        // TBC with the suite to be used
+        sh script: """SUITE=metricbeat DEVELOPER_MODE=false TIMEOUT_FACTOR=3 LOG_LEVEL=TRACE make -C e2e functional-test""", label: "Run functional tests for ${suite}:${tags}"
```

Comment:
```suggestion
        sh script: """SUITE="${suite}" TAGS="${tags}" DEVELOPER_MODE=false TIMEOUT_FACTOR=3 LOG_LEVEL=TRACE make -C e2e functional-test""", label: "Run functional tests for ${suite}:${tags}"
```

Reply:
We need to discuss about this, its' only for testing purposes now, but need to find a way to simplify the commands with a simplistic make goal or something like that.